src/Simulation/Core/Submitters/IQuantumMachineExecutionContext.cs

```diff
@@ -1,11 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+
 namespace Microsoft.Quantum.Runtime
 {
     /// <summary>
     /// Interface to provide configuration details to manage execution.
     /// </summary>
+    [Obsolete("No longer used.")]
```

Comment:
I general it would be nice to say what replaces it. And by replacement I don't always mean direct replacement. If we switched to different way of doing things, it's enough to mention this new way of doing things.

Reply:
Added more specific information about a possible replacement.